.github/workflows/ci-test.yml

```diff
@@ -495,7 +495,7 @@ jobs:
           restore-keys: "crates-${{ runner.os }}"
       - name: run unit tests
         run: |
-          $pre_command && cargo nextest --jobs ${max_threads} --unit --changed-since "origin/$TARGET_BRANCH"
+          $pre_command && mkdir -p target/junit-reports && cargo nextest --jobs ${max_threads} --unit --failure-output=immediate-final --changed-since "origin/$TARGET_BRANCH" --junit target/junit-reports/unit-test.xml
```

Comment:
the mkdir should probably be handled in the reporting logic, but not crucial.

Reply:
Ah you mean in nextest? Hmm.